### PR TITLE
Feat: Added the Image Component

### PR DESCRIPTION
**Pull request by hussiantabha**
Added the responsive and the rounded Images

### CODE REVIEW

File: components/images.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.img-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+.img-responsive-container{
+    display: flex;
```
Comment (Manju-sree) [inline components/images.css]: Try to combine classes in to one when you have same property to use. Like here display: flex

**Review by Manju-sree — APPROVED** [review]
Need small changes as commented other than that code looks good

File: docs.html [hunk 2]
```diff
@@ -169,8 +170,8 @@ <h2>Floating Button</h2>
                     sandbox="allow-scripts allow-same-origin">
               </iframe>
               </div>
-          </div>
-          <div class="display-code-content-card" id="cards">
+          </section>
+          <section class="display-code-content-card" id="cards">
```
Comment (tusharkandpal) [inline docs.html:174]: I think you can optimize this class name.

File: docs.html [hunk 3]
```diff
@@ -25,24 +25,25 @@ <h2>Anti-Strap UI</h2>
             <a href="#badges" class="sidebar-links">Badge</a>
```
Comment (tusharkandpal) [inline docs.html]: should go with 'Badges' here

File: app.css [hunk 4]
```diff
@@ -3,6 +3,7 @@
 @import "./components/badges.css";
 @import "./components/button.css";
```
Comment (tusharkandpal) [inline app.css]: Be consistent with naming forms

**Review by tusharkandpal — CHANGES_REQUESTED** [review]

**Review by tusharkandpal — APPROVED** [review]
👍

**Pull request merged**